### PR TITLE
Added TcsEmotion ad support to amp-ad

### PR DESCRIPTION
**Pull request by TcsEmotion**
# TcsEmotion amp-ad support 

## Request to approve our ad network for amp-ad tag
- we developed an amp-ad network ad tag , this is our first contribution towards an open source project so please review our changes 

✨ New feature

### CODE REVIEW

File: ads/tcsemotion.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
```
Comment (torch2424) [inline ads/tcsemotion.js]: nit: Extra new line 😄

File: ads/tcsemotion.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+
```
Comment (torch2424) [inline ads/tcsemotion.md]: nit: One too many initial new lines 👍

File: ads/tcsemotion.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+## Configuration
+
+Please refer to [Tcsemotion](https://tcsemotion.com/) for more
+information on how to get zone id and other details
+
+### Required parameters
+
+-  `data-zone`: the id of the zone provided by adserver
+-  `data-delhost`: target domain url
+
```
Comment (torch2424) [inline ads/tcsemotion.md]: nit: Extra newline 😄

File: ads/tcsemotion.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {validateData,writeScript} from '../3p/3p';
+
+
```
Comment (torch2424) [inline ads/tcsemotion.js]: nit: Extra new line 😄

File: examples/ads.amp.html [hunk 5]
```diff
@@ -1949,5 +1959,6 @@ <h2>Zucks</h2>
       data-frame-id="_bda46cf5ac">
   </amp-ad>
 
+
```
Comment (torch2424) [inline examples/ads.amp.html]: nit: Extra new line. 😄

File: package-lock.json [hunk 6]
```diff
@@ -0,0 +1,1057 @@
+{
```
Comment (torch2424) [inline package-lock.json]: `package-lock.json` Change should not be in this PR. Can you remove this change? 😄

File: examples/ads.amp.html (hunk 5 above)
Comment (TcsEmotion) [reply examples/ads.amp.html]: sorry sir, i will remove those new lines and push those files

**Comment by TcsEmotion** [comment]
@torch2424  thanks for review and approving our ad network , now i merged my changes into latest master 
@googlebot  I signed it!

**Comment by torch2424** [comment]
@TcsEmotion Thank you for doing that! Been quite busy lately, but I will do one last look over and merge this in. Thank you! 😄

**Comment by TcsEmotion** [comment]
@torch2424 thank you for reviewing our code, we are waiting for your response

**Comment by torch2424** [comment]
@TcsEmotion Sorry for the delay! Re-ran the travis, and integration tests passed. Thank you very much for the contribution 😄

**Comment by Anish-Mohan** [comment]
@torch2424  thanks you very much for your support and review.

**Comment by TcsEmotion** [comment]
@torch2424  thank you very much for your support and review.

**Pull request merged**